openfisca_core/simulations.py

```diff
@@ -114,35 +129,299 @@ def data_storage_dir(self):
                 ).format(self._data_storage_dir).encode('utf-8'))
         return self._data_storage_dir
 
-    @property
-    def holder_by_name(self):
-        warnings.warn(
-            u"The simulation.holder_by_name attribute has been deprecated. "
-            u"Please use entity.get_holder instead. "
-            u"Using simulation.holder_by_name may negatively impact performances",
-            Warning
-            )
+    # ----- Methods to access parameters ----- #
 
-        result = {}
-        for entity in self.entities.itervalues():
-            result.update(entity._holders)
-        return result
+    def parameters_at(self, instant, use_baseline = False):
```

Comment:
> Next 3 methods have just been moved and not edited.

Reply:
There are actually redundant with other methods in `TaxBenefitSystem`. Removing them.